reference-data/api/src/main/java/org/bahmni/module/referencedata/labconcepts/mapper/AttributeResourceMapper.java

```diff
@@ -0,0 +1,27 @@
+package org.bahmni.module.referencedata.labconcepts.mapper;
+
+import org.openmrs.Concept;
+import org.bahmni.module.referencedata.labconcepts.contract.Resource;
+
+import java.util.HashMap;
+
+public class AttributeResourceMapper extends ResourceMapper {
```

Comment:
IMO, should be named "AttributableResourceMapper".  Otherwise the name indicates a AttributeResource. Thats not what we mean - right?